nimlsp.nimble

```diff
@@ -21,8 +22,7 @@ task debug, "Builds the language server":
   exec "nim c --threads:on -d:nimcore -d:nimsuggest -d:debugCommunication -d:debugLogging -o:nimlsp src/nimlsp"
 
 before test:
-  if not fileExists("nimlsp"):
```

Comment:
Why did you remove this check?

Reply:
when I modified I need build first then run test, otherwise the nimlsp is use previous build version, that make me confusing. I thought I just need run nimble test.

Reply:
Aah, that makes sense